package.json

```diff
@@ -9,15 +9,31 @@
     "web": "expo start --web"
   },
   "dependencies": {
+    "@react-native-picker/picker": "^2.4.8",
+    "@react-navigation/bottom-tabs": "^6.5.2",
     "@react-navigation/native": "^6.0.16",
     "@react-navigation/native-stack": "^6.9.4",
-    "expo": "~47.0.8",
+    "axios": "^1.2.1",
+    "expo": "^47.0.9",
+    "expo-checkbox": "~2.2.2",
     "expo-status-bar": "~1.4.2",
+    "formik": "^2.2.9",
+    "native-base": "^3.4.25",
     "react": "18.1.0",
+    "react-hook-form": "^7.40.0",
     "react-native": "0.70.5",
+    "react-native-alert-notification": "^0.3.4",
+    "react-native-android-checkbox": "^3.1.1",
+    "react-native-dropdown-select-list": "^2.0.4",
+    "react-native-gesture-handler": "^2.8.0",
+    "react-native-input-select": "^0.18.0",
     "react-native-safe-area-context": "^4.4.1",
     "react-native-screens": "^3.18.2",
-    "react-router-native": "^6.4.4"
+    "react-native-svg": "13.4.0",
+    "react-native-sweet-alert": "^3.5.0",
+    "react-native-web": "^0.18.10",
+    "react-router-native": "^6.4.4",
+    "sweetalert2": "^11.4.8"
```

Comment:
Are we using all the new packages added?
If not, remove the packages that are not used